packages/contract-wrappers/src/contract_wrappers/exchange_wrapper.ts

```diff
@@ -54,7 +55,7 @@ interface ExchangeContractErrCodesToMsgs {
  */
 export class ExchangeWrapper extends ContractWrapper {
     private _exchangeContractIfExists?: ExchangeContract;
-    private _orderValidationUtils: OrderValidationUtils;
+    private _orderValidationUtilsIfExists?: OrderValidationUtils;
```

Comment:
I don't understand the need for that change. Please elaborate. Makes the code bigger & more complex

Reply:
We need to get the ExchangeContract (an async call) so we cannot do this in the constructor unfortunately.

Reply:
Why were we able to do that before?

Reply:
We didn't instantiate the ExchangeContract (generated wrapper), but instead passed in `contract-wrappers` ExchangeWrapper.